docs/integrations/data-ingestion/clickpipes/postgres/joins_and_denormalization.md

```diff
@@ -0,0 +1,278 @@
+---
+sidebar_label: 'JOINs and Denormalization'
+description: 'How to optimize JOINs and leverage denormalization to speed up JOINs.'
+slug: /integrations/clickpipes/postgres/joins_and_denormalization
+title: 'JOINs and Denormalization'
+---
+
+import clickpipes_joins_mv from '@site/static/images/integrations/data-ingestion/clickpipes/postgres/postgres-cdc-joins-mv.png';
+import Image from '@theme/IdealImage';
+
+Since Postgres is a relational database, its data model is heavily [normalized](https://en.wikipedia.org/wiki/Database_normalization), often involving hundreds of tables. In ClickHouse, denormalization can be beneficial at times to optimize JOIN performance. This page covers common JOINs tips and best practices to denormalize data.
+
+### Optimizing JOINs {#optimizing-joins}
```

Comment:
We have a guide here thats already dedicated. Ok with a little duplicatiom but point to the guide as well.